ethereumetl/deduplication/redis.py

```diff
@@ -0,0 +1,27 @@
+import os
+import hashlib
+import redis
+from ethereumetl.constants import constants
+
+class RedisConnector:
+    def __init__(self):
+        self.ttl = os.environ['REDIS_MESSAGE_TTL']
+        
+        redis_host = os.environ['REDIS_HOST']
+        redis_port = os.environ['REDIS_PORT']
+        redis_database = os.environ['REDIS_DB']
```

Comment:
use chainId for REDIS_DB  ?

Reply:
sure but lets do it for next chains as for current DEGEN we have already provided its value so if we change in between then it might not be able to do dedup based on keys stored in different database

Also, this is configurable from envs so should be easily doable